src/Ng.TraceListeners/AzureTableStorageStatusTraceListener.cs

```diff
@@ -0,0 +1,134 @@
+        }
+
+        protected override void TraceSimpleEvent(DateTime eventTime, Int32 threadId, TraceEventType eventType, String message)
+        {
+            // Called by Trace.WriteLne(). Ignore.
+        }
+
+        protected override void TraceSimpleEvent(DateTime eventTime, Int32 threadId, TraceEventType eventType, String message, Int32 eventId, TraceEventCache eventCache, String source)
+        {
+            if (_table != null)
+            {
+                if (message.StartsWith("#Status ") || message.StartsWith("#Status:"))
+                {
+                    Status status = new Status(eventTime.ToUniversalTime());
+                    status.Level = eventType.ToString();
+                    status.ThreadId = threadId;
+
+                    string trimmedMessage = message.Substring(8);
+
+                    string[] parts = trimmedMessage.Split(new[] { '#' }, StringSplitOptions.RemoveEmptyEntries);
+
+                    foreach (string part in parts)
+                    {
+                        int propertyNameIndex = part.IndexOf('=');
+
+                        if (propertyNameIndex > 0)
+                        {
+                            string propertyName = part.Substring(0, propertyNameIndex);
+                            string propertyValue = part.Substring(propertyNameIndex + 1);
+
+                            switch (propertyName.ToLowerInvariant())
+                            {
+                                case "activity":
+                                    status.Activity = propertyValue;
+                                    break;
+                                case "state":
+                                    status.State = propertyValue;
+                                    break;
+                                case "result":
+                                    status.Result = propertyValue;
+                                    break;
+                                case "details":
+                                    if (!string.IsNullOrWhiteSpace(status.Details))
+                                    {
+                                        status.Details += "; ";
+                                    }
+
+                                    status.Details += propertyValue;
+                                    break;
+                                default:
+                                    throw new InvalidOperationException($"Unknown trace property name: {propertyName}");
+                            }
+                        }
+                        else
+                        {
+                            if (!string.IsNullOrWhiteSpace(status.Details))
+                            {
+                                status.Details += "; ";
+                            }
+
+                            status.Details += part;
+                        }
+                    }
+
+                    TableOperation operation = TableOperation.Insert(status);
```

Comment:
At some point, we might want to include the logged in username. That way we can filter out the entries which correspond to test runs.